lib/spack/spack/environment.py

```diff
@@ -238,7 +241,7 @@ def apply_modifications(self):
                 x.execute()
 
 
-def concatenate_paths(paths):
+def concatenate_paths(paths, separator=';'):
     """
     Concatenates an iterable of paths into a  string of column separated paths
```

Comment:
Thanks! Fixed, and it's amusing to see that it was off to begin with: 
"column" -> "colon"

On 15 May 2016, at 0:10, Massimiliano Culpo wrote:

>> @@ -238,7 +241,7 @@ def apply_modifications(self):
>>                  x.execute()
>>
>>
>> -def concatenate_paths(paths):
>> +def concatenate_paths(paths, separator=';'):
>>      """
>>      Concatenates an iterable of paths into a  string of column 
>> separated paths
>
> Just a minor thing : the function doesn't match docstring anymore
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/LLNL/spack/pull/955/files/a2197f3a417d63f0f13eac64f016abd0528d7748#r63287309